package.json

```diff
@@ -103,7 +103,7 @@
     "base-64": "0.1.0",
     "buffer": "5.2.1",
     "bugsnag-react-native": "2.10.2",
-    "delay": "4.0.0",
+    "delay": "4.0.1",
```

Comment:
testing comments from VS Code

Reply:
![image](https://user-images.githubusercontent.com/464441/45330268-43782000-b52a-11e8-975f-b42bc8e2882c.png)
 
it works!